hardhat.config.ts

```diff
@@ -99,6 +99,15 @@ const config: DeploymentConfig = {
   },
   solidity: {
     compilers: [
+      {
+        version: "0.8.7",
```

Comment:
not used version

Reply:
Ya, can be removed. Originally this was marked as 8.7 but warnings against using 8.7 popped up.